topologies/datacenter/files/MenuOptions.yaml

```diff
@@ -28,16 +28,22 @@ labconfiglets:
   mlag:
     spine1:
       - "Spine1-MLAG-Lab"
+      - "VLANs"
     spine2:
       - "Spine2-MLAG-Lab"
+      - "VLANs"
     leaf1:
       - "Leaf1-MLAG-Lab"
+      - "VLANs"
     leaf2:
       - "Leaf2-MLAG-Lab"
+      - "VLANs"
     leaf3:
       - "Leaf3-MLAG-Lab"
+      - "VLANs"
     leaf4:
       - "Leaf4-MLAG-Lab"
```

Comment:
Remove `Leaf4-MLAG-Lab` configlet for leaf4.  This configures Leaf4 all the way for the MLAG lab.